### PR TITLE
Add info on API keys to the docs

### PR DESCRIPTION
**Pull request by chrissvo**
Fixes: https://github.com/mgmco/bugs/issues/2060

Changes proposed in this pull request:
 * TSIA

cc @mgmco/developers

### CODE REVIEW

File: _sales_tracking/vouchers.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ title: Vouchers
 position: 6
 right_code: |
   ~~~ bash
-  curl -X GET "https://api.membergetmember.co/sales/callback?source=curl&voucher=VOUCHER&name=NAME&email=EMAIL"
+  curl -X POST "https://api.membergetmember.co/sales/statue?source=curl&voucher=VOUCHER&name=NAME&email=EMAIL&api_key=API_KEY"
```
Comment (mtimofiiv) [inline _sales_tracking/vouchers.md]: `statue`!

Comment (chrissvo) [inline _sales_tracking/vouchers.md]: good catch!

**Comment by harianus** [comment]
Are we really using `POST` requests with `GET` data in the url?

**Comment by mtimofiiv** [comment]
> Are we really using POST requests with GET data in the url?

Agree on this point.

The endpoint accepts both a `GET` or a `POST`. In case of a `POST`, those params can be sent via a JSON object - that's already built in.

**Comment by chrissvo** [comment]
When use `POST` and when use `GET` then? Isn't this an example of a situation where the client is _posting_ information to our API?

**Comment by mtimofiiv** [comment]
We should use `POST`. `GET` is still hanging around from the before times. But @harianus is correct to bring this up since `GET`s are supposed to be idempotent.

**Comment by chrissvo** [comment]
idempotent?

**Comment by mtimofiiv** [comment]
Ya, some gay fancy word for non-modifying and the result always being the same (so stateless).

http://restcookbook.com/HTTP%20Methods/idempotency/

**Pull request merged**